tgui/src/util/byond.js

```diff
@@ -1,16 +1,70 @@
 const encode = encodeURIComponent
 
-// Helper to generate a BYOND href given 'params' as an object (with an optional 'url' for eg winset).
-export function href (params = {}, url = '') {
-  return `byond://${url}?` + Object.keys(params).map(key => `${encode(key)}=${encode(params[key])}`).join('&')
-}
+// Helper to generate a BYOND href given 'params' as an object
+// (with an optional 'url' for eg winset).
+export const href = (url, params = {}) => {
```

Comment:
is there something wrong with `function`?

Reply:
It's a legacy syntax with multiple shortcomings (such as hoisting) and it carries redundant features (such as making classes, for which we have an actual "class" keyword), so I just prefer to not use it at all.

With arrow functions, you can:
- Do currying out of the box, like a => b => c
- Omit returns to save space
- Looks syntactically cleaner and unifies the way you declare variables.

Reply:
I don't have anything against arrow functions but we babel this all the way back to IE8 anyways so I'm not sure if it ends up being better in the end

Reply:
Babel is smart enough to figure out simplest ES5 forms, so no issue there.

I checked with the compiler, e.g.:
```js
// original
const x = value => value + 1
// transformed
var x = function x(value) { ... }
```